test/integration/admin_can_modify_its_account_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class AdminCanModifyItsAccountTest < ActionDispatch::IntegrationTest
+  test "admin can only edit its account information " do
+
+   logged_in_admin
+
+   click_link "Edit Account"
+
+   fill_in "Username", with: "Jhun"
+   fill_in "Password", with: "dana"
+
+   click_button "submit"
+   assert page.has_content?("You have succesfully changed your account information")
+
+
+   assert admin_dashboard_index_path , current_path
+   end
+
+   test "registered user cannot edit user info" do
+     user = User.create(username: "user", password: "password")
+     visit '/login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.